pymodbus/framer/rtu_framer.py

```diff
@@ -167,6 +174,67 @@ def check_frame(self):
             Log.debug("Frame advanced, resetting header!!")
             callback(result)  # defer or push to a thread?
 
+    async def asyncProcess(self, exec_request):
+        """Queue the request and start the worker task if needed."""
+        if self._queue is None:
+            self._queue = asyncio.Queue()
+        if self._worker is None:
+            self._worker = asyncio.create_task(self._worker_task())
```

Comment:
If there is a demand for a constantly running task,, then it needs to be done in init.